packages/client/src/client/components/Toggle.tsx

```diff
@@ -57,7 +57,7 @@ export const Toggle = ({ left, right, onChange, isToggled }: Props) => (
         color={
           isToggled
             ? "Colors/Text/text-quaternary (500)"
-            : "Colors/Text/text-primary_alt"
+            : "Colors/Text/text-white"
```

Comment:
elsewhere we have used just "white" .. what do you think?
(I don't really see the point of indirection for black/white .. when are they ever going to be other than #000 and #FFF?)